dashboard/bin/refresh_pd_workshop_material_orders

```diff
@@ -2,11 +2,18 @@
 require_relative '../../lib/cdo/only_one'
 require_relative '../config/environment'
 
+# Be more tolerant of retriable errors here than the default 2 attempts
+MAX_ATTEMPTS = 10
+
 def main
-  Pd::WorkshopMaterialOrder.active.find_each do |order|
-    # Be more tolerant of retriable errors here than the default 2 attempts
-    order.refresh max_attempts: 10
-    order.save!
+  # First, place orders
+  Pd::WorkshopMaterialOrder.unordered.find_each do |order|
+    order.place_order max_attempts: MAX_ATTEMPTS
+  end
+
+  # Next, refresh status on all active orders
+  Pd::WorkshopMaterialOrder.successfully_ordered.active.find_each do |order|
```

Comment:
Worth logging metrics on failed orders? Also - are there unrecoverable errors where we shouldn't retry?